layouts/partials/article.html

```diff
@@ -5,7 +5,8 @@
 				</header>
 {{ end }}
 {{ if or (eq $.mode "full") (eq $.mode "body") }}
-				{{ .TableOfContents }}
+				{{ $globalShowToc := default true (index .Site.Params "toc") }}
+				{{ if default $globalShowToc (index .Params "toc") }}{{ .TableOfContents }}{{ end }}
```

Comment:
I originally broke it into 2 lines as I was still learning the syntax and couldn't get it to work in 1. I must have missed a closing bracket or something because it works now. I'll push in a minute.

`isset` doesn't seem to be appropriate here, as it would return whether the value is set (so `true`, regardless of whether the value is set to `true` or `false`). Testing seems to indicate `isset` doesn't work. `default` already does the `isset` checking.